templates/patterns/file_translation/file_translation.ui_patterns.yml

```diff
@@ -18,6 +18,13 @@ file_translation:
         size: "150000"
         name: "document.pdf"
         language_code: "en"
+    link_attributes:
```

Comment:
Why in file_teaser patter we added possibility to add additional attributes but in file_translation patter we don't have chance to specify extra_attributes.
I've checked in ewcms but didn't found that it works.

Reply:
I've checked and seems like it is possible to add extra_attributes for file_translation items as well.

Reply:
This is limited by the fact, that `file_translation` has a different definition of the `translations` property. It does not allow to pass any additional data along with the translated file (it just takes a list of `FileValueObject`). So I can't pass the `extra_attributes` too. To change it, i would have to refactor the pattern and all references to it.